src/task/MonitorTask.ts

```diff
@@ -60,6 +63,7 @@ export default class MonitorTask
       { ...connectionHealthPolicyConfiguration },
       this.initialConnectionHealthData.clone()
     );
+    this.resourceMonitor = new DefaultResourceMonitor(context.logger);
   }
 
   removeObserver(): void {
```

Comment:
Just curious but is this called in `CleanStoppedSessionTask`?

Reply:
Yes, the `CleanStoppedSessionTask` calls the `removeObserver` of each task to clean up timers and resources.
https://github.com/aws/amazon-chime-sdk-js/blob/5071197d50eda528c8e99df4eb0fd6a7dc874965/src/task/CleanStoppedSessionTask.ts#L36-L38

Reply:
I need to fix some tests that do not clean up the session. Even after the test completes, this resource interval keeps running to cause a missing `document` error.